web/src/Tests/Header.test.js

```diff
@@ -0,0 +1,63 @@
+import { render, screen } from "@testing-library/react";
+import * as React from "react";
+import Header from "../utils/Components/Header";
+
+const setLocalStorage = (age, input) => {
+  window.localStorage.setItem(age, JSON.stringify(input));
+};
+
+describe("Testing Header component", () => {
+  test("testing components appear on screen", () => {
+    render(<Header />);
+    const logo = screen.getByAltText("ExplaiAI logo");
+    const textLabel = screen.getByText("Like I'm");
+    const buttonText = ["five", "ten", "adult"];
+
+    expect(logo).toBeInTheDocument();
+    expect(textLabel).toBeInTheDocument();
+    buttonText.map((button) => {
+      expect(screen.getByTestId(button)).toBeInTheDocument();
+    });
+  });
+
+  test("correct classes has been assigned", () => {
+    render(<Header />);
+    const buttonText = [
+      { button: "five", class: "header__content__ages__selected" },
+      { button: "ten", class: "header__content__ages__unselected" },
+      { button: "adult", class: "header__content__ages__unselected" },
+    ];
+
+    Object.values(buttonText).map((item) => {
+      expect(screen.getByTestId(item.button)).toHaveClass(item.class);
+    });
```

Comment:
It's a little unusual to call `Object.values` on an array, can we not just iterate through it directly?

```suggestion
    const buttonText = [
      { button: "five", class: "header__content__ages__selected" },
      { button: "ten", class: "header__content__ages__unselected" },
      { button: "adult", class: "header__content__ages__unselected" },
    ];

    buttonText.map((item) => {
      expect(screen.getByTestId(item.button)).toHaveClass(item.class);
    });
```